express-post-json/index.js

```diff
@@ -0,0 +1,24 @@
+const express = require('express');
+const app = express();
+app.listen('3000', () => 'server is listening on port 3000');
+
+let nextId = 1;
+const grades = {};
+
+app.get('/api/grades', (req, res) => {
+  const arr = [];
+  for (const gr in grades) {
+    arr.push(grades[gr]);
+  }
+  res.json(arr);
+});
+
+app.use(express.json());
+
+app.post('/api/grades', (req, res) => {
+  const gr = req.body;
+  gr.id = nextId;
+  grades[nextId] = gr;
+  nextId++;
+  res.status(201).send(grades);
+});
```

Comment:
Almost. When `POST`ing, the server should only respond with the newly-created record, not all of them:

---

![image](https://user-images.githubusercontent.com/16143771/129294331-09eaf73c-7066-4920-9c16-53e32fe8d9c2.png)

Reply:
![image](https://user-images.githubusercontent.com/75178406/130177674-e4bbdc7b-4981-4bc3-be4f-6d1e5f1162d3.png)